test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -14,40 +14,33 @@
 
 namespace ILLink.RoslynAnalyzer.Tests
 {
-	public class TestCaseUtils
+	public abstract class TestCaseUtils
```

Comment:
Why is this now an abstract class?

Reply:
It only contains helpers -- it probably should have been an abstract class in the beginning.

In fact, everything in it is static. It should be a static class, but XUnit requires that all `MemberData` work against members in the same class, so the only way to reuse the functionality is to put it in a base class and use inheritance.